message StuInfo{
	required string stuId = 1;
	required string passwd = 2;
	required int32 seatNo = 3;
	required int32 roomId = 4;
	required string startTime =5;	  
	required string endTime = 6;
	required string date = 7;
}